simapp/app.go

```diff
@@ -509,6 +511,7 @@ func NewSimApp(
 	// create IBC module from bottom to top of stack
 	var transferStack porttypes.IBCModule
 	transferStack = transfer.NewIBCModule(app.TransferKeeper)
+	ibcAppRouter.AddRoute(ibctransfertypes.ModuleName, transferStack)
```

Comment:
I realized my comment was wrong, the wiring here is potentially problematic. Will leave as is, but wanted to mention in case we run into e2e failures